Credify/Credify/WebView/SimpleWebView/SimpleWebViewController.swift

```diff
@@ -0,0 +1,264 @@
+//
+//  SimpleWebViewController.swift
+//  Credify
+//
+//  Created by Gioan Le on 21/07/2022.
+//
+
+import UIKit
+import WebKit
+import SafariServices
+
+/// I create this class to show simple web page only. Don't effect with our logic 
+class SimpleWebViewController: UIViewController {
```

Comment:
What's the difference from the original WebViewController? If this is a subset of the original one, original one can simply inherit this class? I see there are a lot of copy and pasted codes.

Reply:
@ngo275 below are some difference
- It doesn't include any the logic of the communication between web app and native SDK.
- It shows navigation bar. 
- It only load a web page with a url. 
- Don't use customization UI.

Reply:
@ngo275 I fixed it